README.md

```diff
@@ -148,6 +148,22 @@ apt::source { 'puppetlabs':
 },
 ~~~
 
+### Add an Apt source served over HTTPS
+
+~~~puppet
+include apt::transport_https
+
+apt::source { 'nodesource':
+  location => "https://deb.nodesource.com/node_0.10",
+  release  => $::lsbdistcodename,
+  repos    => 'main',
+  key      => {
+    'id'     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    'source' => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key',
+  }
+}
```

Comment:
Yeah. The idea was to submit PRs on the modules I care about… but realistically we would end up with a class for every package. And, to make it work safely, you can't used the `class { '::apt::transport_https': ensure=>latest }` syntax, you have to use `include` and hiera.
 
So. I'll close this. `ensure_package` is still the best solution. :disappointed: